.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,23 +10,66 @@ assignees: ''
 
 The release number or commit hash of the version you're using.
 
-## Operating system & version
+## \[Optional\] Setup information
 
-Ex: Windows 10, Ubuntu 18.04, iOS 14.
+If you are reporting a bug about:
+
+- difficulties getting Bevy to build or run on your machine
+- unusual rendering bugs
+- unusual input bugs
+- hardware specific problems
+
+These issues are often specific to hardware or OS configuration and can be challenging to reproduce.
+As a result, please consider including information about:
+
+- the Rust version you're using
```

Comment:
How often do we need to know the rust version for problems? I have seen it once or twice but then it’s normally someone who avidly tracks nighties who is reporting the issue rather than Simone who is coming to Rust for the first time and it’s just not building. But you’d know better here as I could very well have just completely missed these problems as they’re caught and addressed by others in the community such as yourself. If it didn’t happen often then I’d propose moving it lower down the list.

Reply:
On multiple occasions I have seen someone using an outdated rustc version.

Reply:
Yep, that's exactly the problem I want to nip in the bud.

Reply:
```suggestion
- the Rust version you're using (you can get this by running `cargo --version`)
```